app/.env.example

```
@@ -42,4 +42,7 @@ GITHUB_WEBHOOK_SECRET=placeholder
 NODE_TLS_REJECT_UNAUTHORIZED=0
 
 # MCP Server (disabled by default, enable for MCP-enabled deployments)
-NEXT_PUBLIC_MCP_ENABLED=false
\ No newline at end of file
+NEXT_PUBLIC_MCP_ENABLED=false
+
+# AI test-user broker (optional - enables /api/ai-test-user provisioning in local dev / Vercel preview)
+AI_TEST_USER_BROKER_TOKEN=placeholder
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)